docs/protocol/src/concepts/transactions.md

```diff
@@ -35,7 +35,7 @@ pool](../stake/undelegation.md), consuming delegation tokens from the
 transaction's value balance and producing new notes recording the appropriate
 amount of unbonded stake;
 
-- **Commission** descriptions are used by validators to [sweep commission on
+- **Commission** descriptions are used by validators to [claim commission on
 staking rewards](../stake/validator-rewards.md) into shielded notes,
 adding unbonded stake to the transaction's value balance;
```

Comment:
We actually got rid of these entirely, now the commission is paid out by the chain, so we could delete this bullet point.

Reply:
Got it — removed.